molecule/default/converge.yml

```diff
@@ -16,3 +16,9 @@
           - examples
         # just to test whether this also works
         aiida_data_folder: "/usr/local/share"
+        quantum_espresso_version: "6.5"
+        quantum_espresso_executables:
+          - name: pw
+            folder: "/usr/bin"
+            executable: "pw.x"
```

Comment:
does this executable actually exist?
If not, this should not pass the CI tests.

You can simply take any standard unix executable here (e.g. `diff`)

Reply:
Ah, I guess this is not checked by default because codes can be remote...
Anyhow, I'd prefer to take some fake, but existing executable here, since this should fail at least eventually (with newer aiida versions)

Reply:
> Ah, I guess this is not checked by default because codes can be remote... (?)
Anyhow, I'd prefer to take some fake, but existing executable here, since this should fail at least eventually (with newer aiida versions)

To clarify, I shouldn't check for code existence, but still add a dummy executable because at some point `verdi code setup` will do this check internally?

Reply:
In c58732b I have added the dummy executable